### PR TITLE
Include globalnet 2.0 resources in subctl gather

### PR DESCRIPTION
**Pull request by tpantelis**
ClusterGlobalEgressIP
GlobalEgressIP
GlobalIngressIP

### CODE REVIEW

**Review by sridhargaddam — COMMENTED** [review]
This PR looks good. @tpantelis can you also capture the output of "ipset list" from the Gateway nodes similar to how we are capturing the ipsec-status command here - https://github.com/submariner-io/submariner-operator/blob/0105e4d08ef7236073547f1383566fc76c548f6d/pkg/subctl/cmd/gather/cni.go#L53

Please note, we can capture this output (i.e., ipset list) from the Globalnet Pods only as this binary will be present only on Globalnet container.

File: scripts/kind-e2e/lib_subctl_gather_test.sh [hunk 1]
```diff
@@ -89,18 +92,27 @@ function validate_resource_files() {
...
+    nsarg="-A"
+  fi
 
   if [[ $cluster_name == "" ]]; then
     cluster_name=${cluster}
   fi
 
-  names=$(kubectl get $resource --namespace=$ns $selector -o=jsonpath='{.items..metadata.name}')
+  names=$(kubectl get $resource $nsarg $selector -o=jsonpath='{.items..metadata.name}')
+  namespaces=$(kubectl get $resource $nsarg $selector -o=jsonpath='{.items..metadata.namespace}')
```
Comment (vthapar) [inline scripts/kind-e2e/lib_subctl_gather_test.sh]: This has potential to run into problems where number of entries don't match coz some got deleted between these two `get` commands. e.g. Gather test is run at end of e2e when namespace is deleted and objects are being deleted but not all gone yet. Not sure if we can perfectly address the issue, but would be good to echo some warning message if the no. of items in these two don't match to help troubleshoot any flaky failures.

**Review by vthapar — CHANGES_REQUESTED** [review]

File: scripts/kind-e2e/lib_subctl_gather_test.sh (hunk 1 above)
Comment (tpantelis) [reply scripts/kind-e2e/lib_subctl_gather_test.sh]: I changed it to capture the whole JSON and use `jq` to parse the fields.

**Comment by tpantelis** [comment]
> This PR looks good. @tpantelis can you also capture the output of "ipset list" from the Gateway nodes similar to how we are capturing the ipsec-status command here -
> 

Done.

**Review by sridhargaddam — APPROVED** [review]
Thanks for addressing this use-case @tpantelis

**Review by vthapar — APPROVED** [review]

**Comment by dfarrell07** [comment]
Do we want to go ahead and merged this so it makes it in to 0.10.0-rc1? /cc @mangelajo

**Comment by tpantelis** [comment]
> Do we want to go ahead and merged this so it makes it in to 0.10.0-rc1? /cc @mangelajo

Yes. We need a committer to approve.

**Review by mangelajo — APPROVED** [review]

**Pull request merged**